vagrant.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = ">= 1.3.6"
   s.rubyforge_project         = "vagrant"
 
-  s.add_dependency "bundler", ">= 1.5.2", "< 1.8.0"
+  s.add_dependency "bundler", ">= 1.5.2"#, "< 1.8.0"
```

Comment:
Is the comment intented? E.g. will the constraint be re-added soon or will it be gone for good?